### PR TITLE
Add support for GraphQL

### PR DESCRIPTION
**Pull request by gjtorikian**
Closes https://github.com/github/linguist/issues/2637.

I noticed @charliesome mentioned wanting support for this. Seems like a good idea!

### CODE REVIEW

**Comment by kdaigle** [comment]
😍 Thank you @gjtorikian! You're my only hope.

**Comment by charliesome** [comment]
Oh my 😻

**Comment by gjtorikian** [comment]
@charliesome Yep! I don't know if it's the preferred way but there's some mechanism that converts the CSON grammar so that you can use the same definitions in Linguist and Atom. For example, here's [a recent PR](https://github.com/github/linguist/pull/2937) that also opts to use a language built for Atom.

File: vendor/licenses/grammar/language-graphql.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+type: grammar
+name: language-graphql
+license: mit
+curated: true
+---
+MIT
```
Comment (bkeepers) [inline vendor/licenses/grammar/language-graphql.txt]: This should have attribution (`Copyright 2016 Author McName`) and the full text of the license.

Comment (arfon) [inline vendor/licenses/grammar/language-graphql.txt]: Well perhaps Licensed should have picked that up 😛. Will fix shortly.

Comment (arfon) [inline vendor/licenses/grammar/language-graphql.txt]: Fixed in 5c7aa5406af530d563846ed09d2a3d337eb8fadc

**Comment by steveluscher** [comment]
<3

**Pull request merged**